pkg/fftypes/tokenaccount.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fftypes
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+)
+
+type TokenAccountIdentifier struct {
+	Namespace  string `json:"namespace"`
+	PoolID     *UUID  `json:"poolId,omitempty"`
```

Comment:
Once again, the combo of Namespace+PoolID feels a bit odd... can't decide if this is the best combo of fields to return or not.

Reply:
After talking with others, it does seem we could drop namespace here. Also might want to use the on-chain protocolId instead of the FireFly poolId.